kratos/python_scripts/point_output_process.py

```diff
@@ -89,23 +103,73 @@ def ExecuteInitialize(self):
                 err_msg += 'It can only be double, component or array3d!'
                 raise Exception(err_msg)
 
+        # search needs to happen before first time increment in case process was restarted
+        # otherwise TimeBasedAsciiFileWriterUtility would not get required time step
+        if self.model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED]:
+            self.search_done = True
+            self.__SearchPoint()
+
+    def ExecuteBeforeSolutionLoop(self):
+        pass
+
+    def ExecuteInitializeSolutionStep(self):
+        time = self.model_part.ProcessInfo[KratosMultiphysics.TIME]
+
+        # search for point, moved here, so search happens at start time of the time interval
+        if time >= self.interval[0] and not self.search_done:
```

Comment:
can you pleas use the `IntervalUtility` for this?

Reply:
sure! I hadn't seen it before, it's nice.